onnx/onnx.in.proto

```diff
@@ -311,7 +311,7 @@ message TensorProto {
   repeated int64 dims = 1;
 
   // The data type of the tensor.
-  optional DataType data_type = 2;
+  optional int32 data_type = 2;
```

Comment:
For each of these lines, can we add a comment "// This field MUST have a valid TensorProto.DataType value".
This will at least help capture what a valid ONNX model is.

Reply:
Sure added